climakitae/uncertain_utils.py

```diff
@@ -238,8 +257,10 @@ def grab_temp_data(copt):
 
     Attributes
     ----------
-    copt: CmipOpt
-        Selections for variable, area_subset, location, area_average, timescale
+    copt: object
+        Selections: variable, area_subset, location, area_average, timescale
```

Comment:
The correct object type for the input `copt` is `CmipOpt`, not `object`, because the function requires the input of the `CmipOpt` object. Hopefully that makes sense? So it should be `copt: CmipOpt`